### PR TITLE
Reduce bundlesize

### PR DESCRIPTION
**Pull request by jorisre**
If you prefer to keep readability vs bundlesize then let me know ;) I can revert some parts

### CODE REVIEW

File: src/logic/getCheckboxValue.ts [hunk 1]
```diff
@@ -17,18 +17,16 @@ export default (options?: HTMLInputElement[]): CheckboxFieldResult => {
...
-        .map(({ value }) => value);
+        .map((option) => option.value);
       return { value: values, isValid: !!values.length };
     }
 
-    const { checked, value, attributes } = options[0];
-
-    return checked
+    return options[0].checked
       ? // @ts-expect-error expected to work in the browser
```
Comment (jorisre) [inline src/logic/getCheckboxValue.ts:25]: Need help for this one, here is the TypeScript error
Screen Shot 2021-03-05 at 17 13 05

**Comment by bluebill1049** [comment]
That's amazing!

File: src/useFieldArray.ts [hunk 2]
```diff
@@ -130,18 +132,15 @@ export const useFieldArray = <
...
-    return `${name}.${index}`;
+    return options
+      ? !isUndefined(options.focusIndex)
+        ? `${name}.${options.focusIndex}`
+        : options.focusName
+        ? options.focusName
+        : !options.shouldFocus
+        ? ''
+        : `${name}.${index}`
+      : `${name}.${index}`;
```
Comment (bluebill1049) [inline src/useFieldArray.ts:145]: Awesome!!

**Comment by bluebill1049** [comment]
so many optimization in here! it's so great.

**Pull request merged**